src/zajecia18/zadanie1/Main.java

```diff
@@ -0,0 +1,14 @@
+package zajecia18.zadanie1;
+
+import java.util.stream.Stream;
+
+public class Main {
+    public static void main(String[] args) {
+        Stream.iterate(8, n -> n + 1)
```

Comment:
Proponowałbym nadawać lepsze nazwy zmiennych np `number` zamiast `n` pozwoli to na dużo szybsze zrozumienie czym jest pojedyńczy element w danym etapie.